package.json

```diff
@@ -72,7 +72,8 @@
     "**/@typescript-eslint/eslint-plugin": "^4.5.0",
     "**/@typescript-eslint/parser": "^4.5.0",
     "webpack": "4.44.2",
-    "react-error-overlay": "6.0.9"
+    "react-error-overlay": "6.0.9",
+    "babel-loader": "8.1.0"
```

Comment:
It might be worth trying to remove this resolution and seeing if stuff builds - I had also installed `ts-node` which might have caused a babel conflict here (thus the resolution), but realized I didn’t need it and remove.